qemu/lib/iozone.py

```diff
@@ -0,0 +1,110 @@
+
+        :param mountpoint: mountpoint/disk letter assigned to disk under test
+        :return: iozone run status and result.
+        """
+        #run iozone on the same device in parallel is not supported
+        test_cmd = test_cmd % mountpoint
+
+        status, output = self.session.cmd_status_output(test_cmd,
+                                                        timeout=self.timeout)
+        if self.params.get("record_results", "yes"):
+            test_disk = mountpoint.split("/")[-1]
+            self.results_path = os.path.join(self.test.resultsdir,
+                                             'iozone_result_%s' % test_disk)
+            logging.debug("Write results to %s" % self.results_path)
+            try:
+                with open(self.results_path, 'a') as iozone_result:
+                    iozone_result.write(output)
+            except:
+                self.results_path = None
+                raise exceptions.TestFail("Failed to write iozone test results to file")
+
+            if self.params.get("post_results", "no") == "yes" and self.results_path:
+                self.post_result(test_disk)
+
+        return status, output
+
+    def post_result(self, test_disk):
+        """
+        Post result after iozone test, generate a series graphs
+        """
+        # use this function, params "-i" for iozone is not suggested.
+        # suggest usage:iozone.exe -azR -r 64k -n 125M -g 512M -M -f I:\testfile".
+        # Otherwise, your result can't be analysed correctly.
+        logging.info("Generate graph of test result")
+        analysisdir = os.path.join(self.test.resultsdir, 'iozone_result_analysis_%s'
+                                   % test_disk)
+        iozone_analyzer = postprocess_iozone.IOzoneAnalyzer(list_files=[self.results_path],
+                                                            output_dir=analysisdir)
+        iozone_analyzer.analyze()
+        iozone_plotter = postprocess_iozone.IOzonePlotter(results_file=self.results_path,
+                                                          output_dir=analysisdir)
+        iozone_plotter.plot_all()
+
+    def clean(self):
+        """
+        Clean old iozone packet after test in guest
+        """
+        clean_cmd = self.params.get("clean_cmd")
+        if clean_cmd:
+            iozone_running_status = utils_misc.wait_for(lambda: "iozone" not in self.session.cmd_output
+                                                        (self.params.get("iozone_pid_check")), self.timeout,
+                                                        0, 5)
+            if not iozone_running_status:
+                logging.warning("Iozone is still running, force to kill it")
+                self.session.cmd(self.params.get("iozone_stop_cmd"))
+            status, output = self.session.cmd_status_output(clean_cmd, timeout=self.timeout)
+            if status:
+                raise exceptions.TestError("Failed to cleanup iozone, output: %r" % output)
```

Comment:
You have multiple lines exceed 79 characters.
And you need to add a new empty line. Not sure why CI did find these issues.